examples/vanillajs/test/SpecRunner.html

```diff
@@ -3,9 +3,9 @@
 <head>
   <title>Jasmine Spec Runner</title>
 
-  <link rel="stylesheet" type="text/css" href="../bower_components/jasmine/lib/jasmine-core/jasmine.css">
-  <script type="text/javascript" src="../bower_components/jasmine/lib/jasmine-core/jasmine.js"></script>
-  <script type="text/javascript" src="../bower_components/jasmine/lib/jasmine-core/jasmine-html.js"></script>
+  <link rel="stylesheet" type="text/css" href="../node_modules/jasmine/lib/jasmine-core/jasmine.css">
+  <script type="text/javascript" src="../node_modules/jasmine/lib/jasmine-core/jasmine.js"></script>
+  <script type="text/javascript" src="../node_modules/jasmine/lib/jasmine-core/jasmine-html.js"></script>
```

Comment:
not yours, but since you're chaning. the `type` attribute on these can be removed.